frontend/src/components/atoms/inputs/DateInput.js

```diff
@@ -1,9 +1,57 @@
-import React from 'react';
-import Input from './Input';
+import { useState } from 'react';
+import Label from './Label';
+import Datetime from 'react-datetime';
+import moment from 'moment';
 
-export const DateInput = (props) => (
-  // the placeholder is used only on Safari and IE11 where input[type="date"] is not implemented.
-  <Input {...props} placeholder={'AAAA-MM-JJ'} type="date" />
-);
+import './DateInput.css';
+import 'react-datetime/css/react-datetime.css';
+
+export const DateInput = ({
+  label = '',
+  helper = '',
+  name = '',
+  placeholder = 'Sélectionnez une date',
+  value = '',
+  disabled = false,
+  onChange,
+  required = false,
+  inputStyle = null,
+  ...rest
+}) => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Datetime
```

Comment:
you should wrap this within a `<div className="fr-input-group">` element to solve your margin issue